library/java/io/envoyproxy/envoymobile/engine/AndroidNetworkMonitor.java

```diff
@@ -20,6 +23,8 @@
  */
 @TargetApi(Build.VERSION_CODES.LOLLIPOP)
 public class AndroidNetworkMonitor extends BroadcastReceiver {
+  private static final String PERMISSION_DENIED_STATS_ELEMENT =
+      "android_access_network_state_denied";
```

Comment:
Maybe something along the lines of

```suggestion
      "android_permissions.network_state_denied";
```

would let us use this namespace to track other instances we (might) care about in the future.